Objects/longobject.c

```diff
@@ -6168,6 +6168,19 @@ long_long_meth(PyObject *self, PyObject *Py_UNUSED(ignored))
     return long_long(self);
 }
 
+/*[clinic input]
+int.is_integer
+
+Returns True.
```

Comment:
```suggestion
Returns True.

Exists for duck type compatibility with `float.is_integer`.
```

I think this should also be in `__doc__`.